prescryptchain/templates/home.html

```diff
@@ -167,7 +167,7 @@ <h3>RECENT TXS</h3>
             chart.series[0].addPoint(point, true, shift);
 
             // call it again after one second
-            setTimeout(requestData, 1000);
+            setTimeout(requestData, 100000);
```

Comment:
This is a temporary fix, @imoutaharik debemos retomar cambiar la grafica! Cuando tengas un tiempo lo platicamos